packages/adminrouter/extra/src/README.md

```diff
@@ -32,6 +32,71 @@ The endpoint should only use relative links for links and referenced assets such
 
 Tasks running in nested [Marathon app groups](https://mesosphere.github.io/marathon/docs/application-groups.html) will be available only using their service name (i.e, `<dcos-cluster>/service/<service-name>`) and not considering the marathon app group name (i.e., `<dcos-cluster>/service/app-group/<service-name>`).
 
+## Authorization and authentication
+
+DC/OS uses JWT at the core of its authentication and authorization features and
+Admin Router performs a vital role in facililating it. Lots of services which
+constite DC/OS do not have their onw authorizers and rely on Admin Router to
+provide them with authn/authn+authz enforcement.
+
+Authn and authz code differes considerably between Open Source and Enterprise
+versions of Admin Router. This documentation focuses on describing common and
+Open Source related code. For EE features documentation please consult
+README.md in EE Admin Router repository.
+
+### Authentication
+
+In order for Admin Router to establish identity of the principal issuing a
```

Comment:
Are we ready to introduce a `principal` concept to our documentation? So far there are no mentions of this term in the `dcos` repo.

```
$ dcos (master)$ grep "principal" . -R
./packages/marathon/build:    --mesos_authentication_principal "dcos_marathon" \\
```